README.md

```diff
@@ -120,12 +120,8 @@ const Resolver = new Resolver(<ipfs-block-service instance>)
 
 ## Packages
 
-Listing of dependencies from the IPFS ecosystem.
+Listing of dependencies from the IPLD ecosystem.
```

Comment:
Can you please add a comment on how this table is generated?

Reply:
done. See below